crates/cxx-qt-gen/src/parser/property.rs

```diff
@@ -29,6 +32,10 @@ pub struct QPropertyFlags {
     pub(crate) read: FlagState,
     pub(crate) write: Option<FlagState>,
     pub(crate) notify: Option<FlagState>,
+    pub(crate) reset: Option<Ident>, // TODO: in future might be able to generate the function if T has a default
+    pub(crate) is_final: bool, // TODO: `final` is a keyword, see https://github.com/KDAB/cxx-qt/issues/1002
```

Comment:
fun, lets discuss in the issue

Reply:
Consensus seems to be to use capitals for all of them for now to match Qt, so `READ`, `WRITE`, `FINAL` etc

Reply:
the TODO comment for is_final could be removed here

Reply:
done